src/app/hearing/models/possible-action-config.ts

```diff
@@ -4,6 +4,6 @@ import { Observable } from 'rxjs';
 export interface PossibleActionConfig {
     enabled: boolean,
     openDialog: () => Observable<any>,
-    callService: (hearing: Hearing) => void,
+    callService: (hearing: Hearing, description?: string) => void,
```

Comment:
Refers to a lower comment: Change description to: 'data: any' to make it more generic.